lib/src/speed_dial.dart

```diff
@@ -56,29 +57,32 @@ class SpeedDial extends StatefulWidget {
   /// The speed of the animation
   final int animationSpeed;
 
-  SpeedDial({
-    this.children = const [],
-    this.visible = true,
-    this.backgroundColor,
-    this.foregroundColor,
-    this.elevation = 6.0,
-    this.overlayOpacity = 0.8,
-    this.overlayColor = Colors.white,
-    this.tooltip,
-    this.heroTag,
-    this.animatedIcon,
-    this.animatedIconTheme,
-    this.child,
-    this.marginBottom = 16,
-    this.marginRight = 16,
-    this.onOpen,
-    this.onClose,
-    this.closeManually = false,
-    this.shape = const CircleBorder(),
-    this.curve = Curves.linear,
-    this.onPress,
-    this.animationSpeed = 150
-  });
+  /// The orientation of the children. Default is [SpeedDialOrientation.Up]
+  final SpeedDialOrientation orientation;
+
+  SpeedDial(
```

Comment:
Please fix formatting here, should be like it was before.

Reply:
Done. Whoopsie! :-)